yaptide/routes.py

```diff
@@ -27,7 +26,8 @@ def decorator(f):
         """Determines if the access or refresh token is valid"""
         @wraps(f)
         def wrapper(*args, **kwargs):
-            token: str = request.cookies.get('refresh_token' if isRefresh else 'access_token')
+            token: str = request.cookies.get(
+                'refresh_token' if isRefresh else 'access_token')
```

Comment:
```suggestion
            token: str = request.cookies.get('refresh_token' if isRefresh else 'access_token')
```

Reply:
also `isRefresh` is no pythonic style. What does it mean ? Shouldn't it rather be `is_refreshed` ?